atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1815,15 +1797,6 @@ private void commitWrites(TransactionService transactionService) {
                 if (validationNecessaryForInvolvedTablesOnCommit()) {
                     throwIfImmutableTsOrCommitLocksExpired(null);
                 }
-
-                // if the cache has been used, we must work out which values can be flushed to the central cache by
-                // obtaining a commit update, which is obtained via the get commit timestamp request.
-                if (getCache().hasUpdates()) {
```

Comment:
We no longer require this given that we have `onTransactionCommit`, and that we don't need to construct the read-only cache for serialisable read-only transactions.

Reply:
🌮